libs/coin-modules/coin-aptos/src/__tests__/network/client.test.ts

```diff
@@ -29,7 +45,7 @@ describe("Aptos API", () => {
   beforeEach(() => {
     mockedAptos = jest.mocked(Aptos);
     mockedApolloClient = jest.mocked(ApolloClient);
-    mockedPost = jest.mocked(post);
+    mockedPost = jest.mocked(postAptosFullNode);
```

Comment:
not following these mocks, we can't make calls to the network

Reply:
I replaced the method that was being used, it's not calling the network, it's mocking the return of the method:

`mockedPost.mockImplementation(async () => ({ data: { hash: "ok" } }));`